### ISSUE TITLE
Missing Timer for The Jaws of Undeath

### ISSUE THREAD

**Issue by Webberjo** [ufx/GarlandTools#20]
Both [Garland Bell](https://www.garlandtools.org/bell/) and [FFXIV Fisher](https://www.garlandtools.org/ffxivfisher/) are missing a timer for [The Jaws of Undeath](https://en.ff14angler.com/fish/3193).
According to comments on CBH, the weather condition is Clear Skies/Fair Skies into Clouds/Fog.
